### ISSUE TITLE
Multiple declarations of internal/asm/f64.Ger on amd64 and Go <1.8

### ISSUE THREAD

**Issue by vcabbage** [gonum/gonum#472]
Apologies for not using your issue template. I'm passing along something that came up on Gopher's Slack. Someone was trying to `go get` the package which failed with the error:

```
# gonum.org/v1/gonum/internal/asm/f64
gonum/internal/asm/f64/ge_noasm.go:12: Ger redeclared in this block
        previous declaration at gonum/internal/asm/f64/ge_amd64.go:12
```

https://github.com/gonum/gonum/blob/73e313d592593d4180adbb4a164b4d100bc1066b/internal/asm/f64/ge_noasm.go#L5

https://github.com/gonum/gonum/blob/73e313d592593d4180adbb4a164b4d100bc1066b/internal/asm/f64/ge_amd64.go#L5

The build tags allow both files to be included in the build when on amd64 and using Go <1.8.

The person experiencing the issue was able to upgrade from Go 1.6 to 1.10 to avoid the issue, but I wanted to let you know about it.

**Comment by chewxy** [comment]
Is the reason for 1.8+ because of the new instructions added?

**Comment by kortschak** [comment]
Yes, `MOVDDUP`, but generally even without the breaking of old Go versions, we don't guarantee that more than 3 versions of Go will work - the list of supported versions can be found in [.travis.yml](https://github.com/gonum/gonum/blob/master/.travis.yml#L7-L10).

**Comment by chewxy** [comment]
Can I suggest supporting at least back until 1.6? The main reason for this is the Go repo in apt for debian and ubuntu is 1.6.2. I recently had to [support that](https://twitter.com/chewxy/status/985313556546506752) and for many people using Go is a matter of `sudo apt install go`. Ubuntu is also by far the most popular linux distro for a lot of things, so it might be worth looking into.

**Comment by kortschak** [comment]
No, that's really not an option. Pre-1.8 is already difficult, and the 1.7 changes are fairly wide-spread. We already support more than is standard (current release plus one).

**Comment by chewxy** [comment]
Time to petition the Ubuntu people to get updated then.

**Comment by sbinet** [comment]
or, just use `godeb`:
- https://github.com/niemeyer/godeb
